### ISSUE TITLE
Related selectors on one line

### ISSUE THREAD

**Issue by jehoshua02** [deseretdigital-ui/CSS-Guidelines#6]
In [Anatomy of a Ruleset](https://github.com/deseretdigital-ui/CSS-Guidelines/blob/gh-pages/index.md#anatomy-of-a-ruleset) it is suggested to place related selectors on the same line. I would like to have each selector on it's own line. But they should still be ordered by relation.

Before Closing Issue:
- [x] Response from @jstoutenburg
- [x] Response from @janiv
- [x] Response from @kbratko
- [x] Response from @jspencerDDM
- [x] Consensus or Decision
- [ ] Pull request or other action if necessary

**Comment by rsahlstrom** [comment]
I'd vote on selector per line. Makes for git diff's easier to see what's going on with each line and what's changing, etc. @deseretdigital-ui/front-end-developers

**Comment by kbratko** [comment]
I agree with Josh. Because related selectors can have very long names for example using BEM.

**Comment by hhatfield** [comment]
When I look at the example, I feel that the `foo`s are supposed to be taken as one set and the `baz` as another. This is explicitly connoted by having them on two separate lines. If `foo-bar` were on a separate line, I don't know if that would mean there are now three sets that happen to share the same rules, or still just two.

It seems that if I needed to make a change to the ruleset that would require me to break apart the sets, it's easier if they're each on one line.
